sklearn/svm/tests/test_sparse.py

```diff
@@ -111,10 +111,19 @@ def test_unsorted_indices():
     # make sure dense and sparse SVM give the same result
     assert_array_almost_equal(coef_dense, coef_sorted.toarray())
 
-    X_sparse_unsorted = X_sparse[np.arange(X.shape[0])]
-    X_test_unsorted = X_test[np.arange(X_test.shape[0])]
+    # scramble the indices
+    def scramble_indices(X):
```

Comment:
This is really reversing each row's indices. I tend to think of "scramble" as more of a random shuffle.

Reply:
I agree. But at the same time it doesn't seem like we're trying a whole bunch of different permutations. Ideally I wish it was more like a few random permutations. But that may be a different PR.